RSA/rsa-securid/_meta/fields.yml

```
@@ -0,0 +1,102 @@
+  activity over multiple SecureID events."
+  type: keywords
+
+rsa.securid.source.id:
+  name: rsa.securid.source.id
+  description: "represents the unique ID of the source device or system that generated the SecureID event."
+  type: keywords
+
+rsa.securid.domain.id:
+  name: rsa.securid.domain.id
+  description: "represents the unique ID of the domain or realm associated with a SecureID event."
+  type: keywords
+
+rsa.securid.agent.domain.id:
+  name: rsa.securid.agent.domain.id
+  description: "This field is similar to rsa.secureid.domain.id, 
+  but it specifically represents the domain or realm associated with the agent (such as a server or application) 
+  that generated the SecureID event."
+  type: keywords
+
+rsa.securid.agent.ip:
+  name: rsa.securid.agent.ip
+  description: "This field represents the IP address of the agent (server or application) 
+  that generated the SecureID event."
+  type: keywords
+
+rsa.securid.policy.method.id:
+  name: rsa.securid.policy.method.id
+  description: "represents the unique ID of the policy method used in a SecureID event. 
+  It can be used to track and analyze policy enforcement activity."
+  type: keywords
+
+rsa.securid.policy.method.name:
+  name: rsa.securid.policy.method.name
+  description: "represents the name of the policy method used in a SecureID event."
+  type: keywords
+
+rsa.securid.policy.id:
+  name: rsa.securid.policy.id
+  description: "represents the unique ID of the policy associated with a SecureID event."
+  type: keywords
+
+rsa.securid.policy.expression:
+  name: rsa.securid.policy.expression
+  description: "represents the expression used in the policy associated with a SecureID event."
+  type: keywords
+
+rsa.securid.objects.type:
+  name: rsa.securid.objects.type
+  description: "represents the type of object (such as user, group, or resource) associated with a SecureID event."
+  type: keywords
+
+rsa.securid.objects.id:
+  name: rsa.securid.objects.id
+  description: "represents the unique ID of the object associated with a SecureID event."
+  type: keywords
+
+rsa.securid.objects.source.id:
+  name: rsa.securid.objects.source.id
+  description: "represents the unique ID of the source device or system associated with the object in a SecureID event."
+  type: keywords
+
+rsa.securid.objects.security.id:
+  name: rsa.securid.objects.security.id
+  description: "represents the unique ID of the security context associated with the object in a SecureID event."
+  type: keywords
+
+rsa.securid.objects.name:
+  name: rsa.securid.objects.name
+  description: "represents the name of the object associated with a SecureID event."
+  type: keywords
+
+user.firstname:
+  name: user.firstname
+  description: ''
+  type: keywords
+
+user.lastname:
+  name: user.lastname
+  description: ''
+  type: keywords
+
+process.env_vars:
+  name: process.env_vars
+  description: ''
+  type: keywords
```

All fields declared in the taxonomy must be prefixed.
```suggestion
rsa.securid.user.firstname:
  name: rsa.securid.user.firstname
  description: ''
  type: keywords

rsa.securid.user.lastname:
  name: rsa.securid.user.lastname
  description: ''
  type: keywords

rsa.securid.process.env_vars:
  name: rsa.securid.process.env_vars
  description: ''
  type: keywords
```
Don't forget to update the parser and the smart-description according to these changes.